docs/components/bases/CodeGroup.vue

```diff
@@ -31,7 +29,7 @@ export default {
     }
   },
   created () {
-    this.$slots.default.map((slot) => {
+    this.$slots.default.filter(slot => !!slot.componentOptions).map((slot) => {
```

Comment:
Better to use `Boolean(slot.componentOptions)`

Reply:
👍